apps/expo/src/app/user/account-settings.tsx

```diff
@@ -76,90 +23,86 @@ const styles = StyleSheet.create({
     flex: 4,
     margin: 4,
     alignSelf: 'center',
-    color: colors.grey
+    color: colors.silver
   },
   navigationListItemChevron: {
     alignSelf: 'center',
-    color: colors.grey
-  },
-  nudgeBtn: {
-    margin: 12
+    color: colors.silver
   }
-})
+});
 
-interface NavigationListItem {
-  title: string,
-  iconName: string,
-  onPress: () => void;
-}
+const tailwindClasses = {
+  mainTile: 'm-4 p-4 rounded-lg bg-bb-dark-gray',
+  mainTileTitle: 'text-2xl text-slate-200',
+  mainTileItem: 'flex-row',
+  navigationListItemLabel: 'flex-auto m-2 self-center text-slate-200',
+  navigationListItemChevron: 'self-center text-slate-200'
+};
+
+const accountItems: NavigationListItem[] = [
+  { title: 'Personal Data', iconName: 'user', onPress: () => router.push('/user/personal-data') },
+  { title: 'Achievements', iconName: 'award', onPress: () => router.push('/user/achievements') },
+  { title: 'Activity History', iconName: 'history', onPress: () => router.push('/user/activity-history') },
+  { title: 'Workout Progress', iconName: 'chart-pie', onPress: () => router.push('/user/workout-progress') }
+];
+
+const otherItems: NavigationListItem[] = [
+  { title: 'Contact Us', iconName: 'envelope', onPress: () => router.push('/contact-us') },
+  { title: 'Privacy Policy', iconName: 'shield-alt', onPress: () => router.push('/privacy-policy') }
+];
 
 const AccountSettings = () => {
   const [notificationsSwitchEnabled, setIsEnabled] = useState(false);
   const toggleNotificationSwitch = () => setIsEnabled(previousState => !previousState);
-
   const { data } = api.user.byId.useQuery({ id: 1 });
   const user = data; // alias for readability
```

Comment:
TIP: You can remove this line if you do the renaming in the line above, example `const { data: user } = api.user.byId.useQuery({ id: 1 });` This will rename the data property to user.